ftests/explorer-ftests-nomode/src/test/java/org/nuxeo/functionaltests/explorer/nomode/ITExplorerTest.java

```diff
@@ -231,6 +233,34 @@ public void testComponentsAlternative() {
         apage.checkAlternative();
     }
 
+    protected void checkOverridePage(String url, String referenceFilePath) {
+        driver.get(NUXEO_URL + url);
+        try {
+            assertEquals(AbstractExplorerTest.getReferenceContent(Paths.get(referenceFilePath)),
+                    driver.getPageSource());
+        } catch (IOException e) {
+            fail(e.getMessage());
```

Comment:
Maybe instead `throw new AssertionError(e)` to get a stack trace.

Reply:
:+1: handled the use case where file is not there at the same time